guide/guide/.js/src/main/scala/io/udash/web/guide/views/ext/demo/bootstrap/DatePickerDemo.scala

```diff
@@ -50,10 +50,10 @@ object DatePickerDemo extends AutoDemo with CssView {
         events.append(s"Widget change from $oldDate to $date")
     }
 
-    val showButton = UdashButton()("Show")
-    val hideButton = UdashButton()("Hide")
-    val enableButton = UdashButton()("Enable")
-    val disableButton = UdashButton()("Disable")
+    val showButton = UdashButton(options = UdashButtonOptions(color = BootstrapStyles.Color.Secondary))("Show")
+    val hideButton = UdashButton(options = UdashButtonOptions(color = BootstrapStyles.Color.Secondary))("Hide")
+    val enableButton = UdashButton(options = UdashButtonOptions(color = BootstrapStyles.Color.Secondary))("Enable")
+    val disableButton = UdashButton(options = UdashButtonOptions(color = BootstrapStyles.Color.Secondary))("Disable")
```

Comment:
This "changing to secondary" made the demo snippet uglier. Maybe we can live with the default (whatever it is)?